step.yml

```diff
@@ -1,26 +1,57 @@
 ---
-title: Artifact pull
-summary: Step to pull artifacts of a pipeline
+title: Pull Pipeline intermediate files
+summary: Pull intermediate files generated by the Pipeline Builds.
 description: |-
-  The step downloads build artifacts of a pipeline to a local folder.
+  The step downloads intermediate files (build artifacts generated by the pipeline workflows,
+  that are intended to be shared with subsequent workflows of the pipeline) to a local folder.
 
-  By default, all artifacts generated by any workflow of the pipeline are downloaded. This can be limited
-  by setting the `artifact_sources` input variable.
+  These artifacts can be uploaded by the [Deploy to Bitrise.io step](https://www.bitrise.io/integrations/steps/deploy-to-bitrise-io),
+  via specifying the *Files to share between pipeline stages* input.
```

Comment:
Make sure to add this Step after you have uploaded the intermediate files. 
You can upload the intermediate files using the [Deploy to Bitrise.io Step](https://www.bitrise.io/integrations/steps/deploy-to-bitrise-io)'s **Files to share between pipeline stages** input. The directories you specify will be archived and uploaded as a single file.